config-windows.md

```diff
@@ -0,0 +1,99 @@
+# Windows-specific Container Configuration
+
+This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
+
+You can configure a container's resource limits via the `resources` field of the Windows configuration.
+Do not specify `resources` unless required.
+
+
+#### Memory
+
+`memory` is an OPTIONAL configuration for the container's memory usage.
+
+The following parameters can be specified:
+
+* **`limit`** *(uint64, optional)* - sets limit of memory usage in bytes
+
+* **`reservation`** *(uint64, optional)* - sets soft limit of memory usage in bytes
```

Comment:
Memory management is very different between Linux and Windows. Here, it means a guaranteed minimum amount of memory. I used the existing terminology though. Perhaps incorrectly?